.github/workflows/build_test.yaml

```diff
@@ -310,13 +311,16 @@ jobs:
     if: ${{ ! cancelled() && needs.build.result == 'success' }}
 
     strategy:
+      fail-fast: false
       matrix:
+        unstable: [false]
         backend:
           # run CUDA tests on Ruche supercomputer with Singularity
           - name: cuda
             image: nvcc
             runner: [self-hosted, cuda]
             use_singularity: true
+            unstable: true
```

Comment:
Was it meant to override `matrix.unstable` ? If I understand correctly this creates a new variable `matrix.backend.unstable` ?

I think we need an entry for each backend ?

Reply:
Thank you for the comments.

Ah, that is true. We need to introduce `matrix.backend.unstable` rather than `matrix.unstable`